src/components/DataSampleGridView.tsx

```diff
@@ -51,27 +58,39 @@ const DataSamplesGridView = () => {
       (gestures.length === 1 && gestures[0].recordings.length === 0),
     [gestures]
   );
-  const { isOpen, onClose, onOpen } = useDisclosure();
+  const recordingDialogDisclosure = useDisclosure();
   const connectToRecordDialogDisclosure = useDisclosure();
 
   const connection = useConnectActions();
   const { actions } = useConnectionStage();
   const { isConnected } = useConnectionStage();
   const loadProjectInputRef = useRef<LoadProjectInputRef>(null);
 
+  // For adding flashing animation for new recording.
+  const [newRecordingId, setNewRecordingId] = useState<number | undefined>(
+    undefined
+  );
+  const handleRecordingComplete = useCallback((recordingId: number) => {
+    setNewRecordingId(recordingId);
+  }, []);
+
+  const handleOpenRecordingDialog = useCallback(() => {
+    recordingDialogDisclosure.onOpen();
+  }, [recordingDialogDisclosure]);
```

Comment:
Both of these can be replaced by the functions they call.

Reply:
[02fb37b](https://github.com/microbit-foundation/ml-trainer/pull/420/commits/02fb37bbc1160ba277c825902b237131982605a1)